vera/tests/palette.rs

```diff
@@ -82,3 +82,42 @@ fn palette_8bpp_indexed() -> Result<(), Error> {
 
 	Ok(())
 }
+
+#[test]
+fn palette_gimp() -> Result<(), Error> {
+	init_test_logger();
+	// Unclear why the path needs to be different for this test compared to
```

Comment:
`include_bytes` interprets the data in a file as a byte array at compilation time, with the path would be relative to the source file. Here you seem to be loading the file during runtime, so the path is relative to the executable. Tests can be run from different directories, so loading files at runtime can lead to more configuration needs. If you change the `derive_from_gpl` to accept a byte array and do the actual loading outside, this can be made consistent with the rest of the tests.

Reply:
Thanks for the clarification!